### ISSUE TITLE
Need to change the spacy cli train code to save the each iteration training result on to disk for ner training

### ISSUE THREAD

**Issue by YanLiang1102** [oudalab/Arabic-NER#11]
In that way we can see how each iteration enhance the tag and token accuracy since sometimes I can see that the later iteration actually get a worse result...

**Comment by ahalterman** [comment]
Good idea! Or you can do it Keras-style: generate the validation loss/accuracy and only save the model if it outperforms the best previous iteration.
